torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1256,6 +1267,7 @@ def sample_inputs_masked_select(op_info, device, dtype, requires_grad):
                    )),
     # NOTE: the derivative for inplace acosh is not implemented
     UnaryUfuncInfo('acosh',
+                   aliases=('arccosh', ),
```

Comment:
These aliases can have their AliasInfos removed from:

https://github.com/pytorch/pytorch/blob/f7313b310505df5593072d17b7f60f516bcf029a/test/test_op_aliases.py#L36

Reply:
I was thinking that we will remove all AliasInfos from test_op_aliases.py once everything is ported ?